test/test_ptdf.jl

```diff
@@ -250,3 +243,25 @@ end
 
     @test_throws ErrorException ptdf_2 = PTDF(sys5; dist_slack = slack_array)
 end
+
+@testset "Test with brute-force PTDF" begin
```

Comment:
This test is non-sensical.. why would we compare the calculation of the PTDF based on a matrix inversion with the essentially the same inversion but done inside power flows? The Matrices used in the DCPowerFlow in PowerFlows.jl ... 

Further, it makes a redundant dependency between the tests in PowerFlows and the ones in PNM... Now if there is a change that breaks PFS in PNM the CI on PNM will also break. 

Please remove this code

Reply:
This test and the dependency on PowerFlows is removed